server/index.js

```diff
@@ -0,0 +1,110 @@
+const express = require("express");
```

Comment:
If this looks good, I will open a PR in Evelyn's one since she set the socket server up [here](https://github.com/hatchways/live-session-manager/blob/ht-2294-socket-server/socket-server/app.js)